Gems/GraphCanvas/Code/Source/Components/StylingComponent.cpp

```diff
@@ -137,14 +137,15 @@ namespace GraphCanvas
     {
         Styling::SelectorVector selectors = m_coreSelectors;
 
-        QGraphicsItem* root = nullptr;
-        SceneMemberUIRequestBus::EventResult(root, GetEntityId(), &SceneMemberUIRequests::GetRootGraphicsItem);
-
+        selectors.reserve(selectors.size() + m_dynamicSelectors.size());
```

Comment:
There are other selectors after dynamic ones. Reserve the exact size may trigger resizing at:
`selectors.emplace_back(m_hoveredSelector);`
Not sure how many other selectors there are, maybe reserve space for all of them?